docs/guides/05-building/cross-platform.md

```diff
@@ -4,4 +4,145 @@ sidebar_position: 2
+
+## How to use GH Action for Building: a glance at Tauri Action
+
+### Introduction
+
+Tauri Action leverages GitHub Actions to simultaneously build your web application as a Tauri native binary for macOS, Linux, and Windows, and automates putting it up on GitHub releases.
+
+This GitHub Action may also be used as a testing pipeline for your Tauri app, guaranteeing compilation runs fine on all platforms for each pull request sent, even if you don't wish to create a new release.
+
+### How to get started
+
+To set up Tauri Action you must first set up a GitHub repository, you can use this action on a repo that doesn't have Tauri configured since we automatically initialize Tauri before building, and configure it to use your Web artifacts.
+
+Go to the [Tauri Action GitHub page][1]. Then head to GitHub Action on your GitHub project and choose "New workflow", then choose "Set up a workflow yourself". Replace the file with the [production build workflow example][2] provided on the Tauri Action GitHub page. Alternatively, you may set up the workflow based on the example below on this page: [example workflow][6]
+
+### Configuration
+
+- You can configure Tauri with the `configPath`, `distPath` and `iconPath` options.
+- You can run custom Tauri CLI scripts with the `tauriScript` option. So instead of running `yarn tauri build` or `npx tauri build`, we'll execute `${tauriScript}`.
+  - Useful when you need custom build functionality when creating Tauri apps e.g. a `desktop:build` script.
+- When your app isn't on the root of the repo, use the `projectPath` input.
+
+- You may modify the workflow name, change the triggers, and add more steps such as npm run lint or npm run test. The important part is that you keep the:
+
+```yaml
+- uses: tauri-apps/tauri-action@v0
+```
+
+line at the end of the workflow, since this runs the build script and releases the artifacts.
+
+### Please note
+
+- This line next to the end of the workflow:
+
+```yaml
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+```
+
+This GitHub Token is automatically issued by GitHub for each workflow run without further configuration, which means there is no risk of secret leakage. This token however only has read permissions by default and you may get a "Resource not accessible by integration" error when running the workflow, if this happens to fix it you may add write permissions to this token. To do this head over to your Project Settings, and then select Actions, scroll down until "Workflow permissions" and check "Read and write permissions".
+
```

Comment:
```suggestion
:::note

```yaml
env:
  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

The GitHub Token is automatically issued by GitHub for each workflow run without further configuration, which means there is no risk of secret leakage. This token however only has read permissions by default and you may get a "Resource not accessible by integration" error when running the workflow. If this happens you may need to add write permissions to this token. Follow [this GitHub guide](Insert link to GitHub on how to do this) for instructions on how to do this.
```

As a note on the link in the bottom, I'd generally recommend linking out to the respective platform's guide for steps like these in case they change over time